flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala

```diff
@@ -654,17 +654,23 @@ case class WindowAggregate(
     }
 
     // validate property
-    if (propertyExpressions.nonEmpty) {
-      resolvedWindowAggregate.window match {
-        case TumblingGroupWindow(_, _, size) if isRowCountLiteral(size) =>
-          failValidation("Window start and Window end cannot be selected " +
-                           "for a row-count Tumbling window.")
+    propertyExpressions.foreach {
+      _.child match {
+        case WindowEnd(_) | WindowStart(_) =>
+          resolvedWindowAggregate.window match {
+            case TumblingGroupWindow(_, _, size) if isRowCountLiteral(size) =>
+              failValidation(
+                "Window start and Window end cannot be selected " +
+                  "for a row-count Tumbling window.")
 
-        case SlidingGroupWindow(_, _, size, _) if isRowCountLiteral(size) =>
-          failValidation("Window start and Window end cannot be selected " +
-                           "for a row-count Sliding window.")
+            case SlidingGroupWindow(_, _, size, _) if isRowCountLiteral(size) =>
+              failValidation(
+                "Window start and Window end cannot be selected " +
+                  "for a row-count Sliding window.")
 
-        case _ => // ok
+            case _ => // ok
```

Comment:
Can you add a comment here that the `RowtimeAttribute` and `ProctimeAttribute` should pass ?